civihr_employee_portal/src/Helpers/NodeHelper.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Drupal\civihr_employee_portal\Helpers;
+
+class NodeHelper {
+
+  /**
+   * Finds a single node
+   *
+   * @param array $criteria
+   *   The criteria for searching for the node
+   *
+   * @return \stdClass|null
+   *   The node, if found. NULL if not found or more than one exist.
+   */
+  public static function findOneBy($criteria = []) {
```

Comment:
The method name is a bit off from what it does, It sounds like it should return the first node it finds based on the criteria or NULL of it can't find any rather than returning NULL when it finds more than one.
Maybe a `findIfOnlyOne` or `getIfOnlyOne` or `findIfOnlyOneExists` or something similar will be a better name.

Reply:
Good point, maybe I'll just change it to returning the first it finds, or null if it doesn't find any. I was trying to copy [Doctrine's method](http://www.doctrine-project.org/api/orm/2.5/source-class-Doctrine.ORM.EntityRepository.html#191) and they seem to return just the first or NULL.